package.json

```diff
@@ -84,14 +89,23 @@
       "stylelint-no-unsupported-browser-features"
     ],
     "rules": {
+      "string-quotes": "single",
+      "color-hex-length": "long",
```

Comment:
Actually I agree with @delawski that we should not be strict in this case, but the [WordPress config already adds strictness for short](https://github.com/WordPress-Coding-Standards/stylelint-config-wordpress/blob/2a9730719cd569ffbfa468e60997f88ff19c93fa/index.js#L19). I prefer long over short, for consistency.

Opinion:
This...
```
$blue:   #0d2d57;
$white:  #ffffff;
$pink:   #e73064;
$black:  #000;
```
... looks better than this:
```
$blue:   #0d2d57;
$white:  #fff;
$pink:   #e73064;
$black:  #000;
```

But I am happy to go either way. 😉